metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -882,10 +882,31 @@ def get_connection_object_page(
                 error and (error.get(c.EXTENSIONS) or {}).get(c.SEVERITY) == c.WARNING
                 for error in errors
             ):
+                # filter out PERMISSIONS_MODE_SWITCHED to report error in human-readable format
```

Comment:
are the `PERMISSIONS_MODE_SWITCHED` errors marked as severity = warning?

Reply:
Yes, below was the error message in report 

{'message': 'One or more of the attributes used in your filter contain sensitive data so your results have been automatically filtered to contain only the results you have permissions to see', 'extensions': {'severity': 'WARNING', 'code': 'PERMISSIONS_MODE_SWITCHED', 'properties': {'workbooksConnection': ['projectNameWithin']}}}